### PR TITLE
add basic Sidebar styling

### LINKED ISSUE #14: Sidebar styling
See #7 

- [x] Misc visual styling
- [x] Show active state (which page is currently being shown)

### PR DESCRIPTION
**Pull request by rjw57**
Style the sidebar to be correctly positioned and sized. Arrange the menu
as per the IAR design. The navigation is implemented via the Material UI
List component but with a custom SidebarNavLink component which renders
a link to static pages and styles itself appropriately when the currentl
location matches the link.

Closes #14.

### CODE REVIEW

File: src/components/Sidebar.js [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
-      <li><Link to="/assets/dept">My department</Link></li>
-      <li><Link to="/assets/all">All</Link></li>
-      <hr/>
-      <li><Link to="/help">Help</Link></li>
-      <hr/>
-      <li><LogoutLink>Sign out</LogoutLink></li>
-    </ul>
-  </Drawer>
+const Sidebar = ({ classes, history, logout }) => (
+  <div>
```
Comment (joeadams04) [inline src/components/Sidebar.js]: The sidebar height is currently constrained to the height of the Page component. For example when there are no/or little assets in the table.

We could force the Page component to 100% height, however it's probably easier to fix the Sidebar instead if we every need to use the dynamic Page size for styling.

Adding a class to the root of the Sidebar component div with `height: 100vh` should do the trick.

File: src/components/Sidebar.js [hunk 2]
```diff
@@ -1,25 +1,37 @@
...
+    <div className={classes.drawerHeader}>
+      <Toolbar>
+        <Typography variant='title' color='inherit'>IAR</Typography>
```
Comment (joeadams04) [inline src/components/Sidebar.js]: IAR text was a placeholder in the prototype. It's been decided to use the University logo here instead. This logo should also provide a link back to home, in our case, the asset department list.

File: src/components/Sidebar.js (hunk 1 above)
Comment (joeadams04) [inline src/components/Sidebar.js]: The sidebar height is currently constrained to the height of the Page component. For example when there are no/or little assets in the table, we have a very short Sidebar.

We could force the Page component to 100% height, however it's probably easier to fix the Sidebar instead if we ever need to use the dynamic Page size for styling.

Adding a class to the root of the Sidebar component div with `height: 100vh` should do the trick.

**Comment by rjw57** [comment]
@joeadams04 Code review changes in ecbd4f4

**Pull request merged**